ouroboros-consensus-test/test-storage/Test/Ouroboros/Storage/ChainDB/StateMachine.hs

```diff
@@ -736,24 +737,24 @@ lockstep model@Model {..} cmd (At resp) = Event
   where
     (mockResp, dbModel') = step model cmd
     newIters   = RE.fromList $ zip (iters resp) (iters mockResp)
-    newReaders = RE.fromList $ zip (rdrs  resp) (rdrs  mockResp)
+    newFollowers = RE.fromList $ zip (rdrs  resp) (rdrs  mockResp)
```

Comment:
The `=` are out of alignment now.

Reply:
they were never aligned here, I'm gonna skip this one